scene/main/node.cpp

```diff
@@ -2173,11 +2173,6 @@ Node *Node::_duplicate(int p_flags) const {
 Node *Node::duplicate(int p_flags) const {
```

Comment:
I guess we could now keep only `Node::duplicate` and remove the extraneous `Node::_duplicate`.